### PR TITLE
Ingest UIUC claims; add topic information script

### LINKED ISSUE #107: UIUC Claims + ISI SS for Topic/Subtopic/Template matching

### PR DESCRIPTION
**Pull request by joecummings**
Closes #107

- Creates ingester for UIUC claims
- Creates script to add topic information to Claims based on SS
- Refactors Pegasus pipeline code
- Fixes some MyPy and PyLint errors

### CODE REVIEW

**Review by elizlee — CHANGES_REQUESTED** [review]
A few small things to look at

File: cdse_covid/pegasus_pipeline/claim_pipeline.py [hunk 1]
```diff
@@ -36,21 +37,22 @@ def main(params: Parameters) -> None:
     # Base info
     base_locator = Locator(("claims",))
     input_corpus_dir = params.existing_directory("corpus")
+    from_raw_documents = False
```
[the remark this answers is not in the document]
Comment (joecummings) [reply cdse_covid/pegasus_pipeline/claim_pipeline.py:40]: Yes, or automatically set based on if a uiuc claims file is provided.

File: cdse_covid/pegasus_pipeline/claim_pipeline.py [hunk 2]
```diff
@@ -70,6 +72,7 @@ def main(params: Parameters) -> None:
     edl_internal_file = ValueArtifact(value=edl_mapping_file, depends_on=[edl_job])
 
     # AMR parsing over the entirety of each document
+    # TODO: Move this shite
```
[the remark this answers is not in the document]
Comment (joecummings) [reply cdse_covid/pegasus_pipeline/claim_pipeline.py:75]: Oops, should remove this comment. Meant to refactor that code into its own function, as well.

File: cdse_covid/semantic_extraction/run_amr_parsing.py [hunk 3]
```diff
@@ -38,6 +38,7 @@ def main(
     claim_ds = ClaimDataset.load_from_dir(input_dir)
 
     for claim in claim_ds.claims:
+        logging.warning("Processing claim %s", claim.claim_id)
```
[the remark this answers is not in the document]
Comment (joecummings) [reply cdse_covid/semantic_extraction/run_amr_parsing.py:41]: Correct.

File: cdse_covid/pegasus_pipeline/claim_pipeline.py [hunk 4]
```diff
@@ -72,26 +72,21 @@ def main(params: Parameters) -> None:
...
-        input_corpus_dir,
-        amr_params,
-        amr_all_loc,
-        amr_all_python_file,
-        amr_all_output_dir,
-        amr_max_tokens,
-        larger_resource,
-    )
+    if amr_params.optional_existing_file("python_file_all"):
+        _ = amr_over_all_docs(
```
Comment (elizlee) [inline cdse_covid/pegasus_pipeline/claim_pipeline.py:82]: What is the reason for `_ =` here?

File: cdse_covid/semantic_extraction/utils/amr_extraction_utils.py [hunk 5]
```diff
@@ -13,7 +13,24 @@
 from cdse_covid.semantic_extraction.mentions import XVariable
 
 PROPBANK_PATTERN = r"[a-z]*-[0-9]{2}"  # e.g. have-name-91
-STOP_WORDS = set(stopwords.words("english")).union({"like"})
+PRONOUNS = {
```
Comment (joecummings) [inline cdse_covid/semantic_extraction/utils/amr_extraction_utils.py:16]: LMK if you think there are any other pronouns that should be included here.

**Review by elizlee — APPROVED** [review]
Just one question, but LGTM!

File: cdse_covid/pegasus_pipeline/claim_pipeline.py (hunk 4 above)
Comment (joecummings) [reply cdse_covid/pegasus_pipeline/claim_pipeline.py:82]: I get a "unused_variable" error if I assign it to a variable; however, I also get an error if I don't return anything. So the hacky fix is that I return a "_"

**Pull request merged**